### ISSUE TITLE
Clarification on Dataset Used for Image Classification Experiments in Evaluation Using Linear Probing

### ISSUE THREAD

**Issue by Hanminghao** [JiawenChenn/STimage-1K4M#1]
Hello, author. Thank you very much for open-sourcing such excellent work. I would like to ask, in your paper you mentioned that you conducted image classification experiments on the fine-tuned CLIP and PLIP during the **Evaluation using linear probing**, but it seems that you did not specify which dataset these experiments were conducted on (it’s also possible that I might have overlooked this). I would greatly appreciate it if you could clarify this for me.

**Comment by JiawenChenn** [comment]
Hi, thanks for pointing this out. The linear probing (Figure 3) is conducted on the human DLPFC samples. The detail evaluation is as follows: we first fine-tuned the CLIP/PLIP on all human DLPFC sample, then the linear probing is tested on human DLPFC again(we trained logistic regression model on 80% of the DLPFC spots, then choose the hyper-parameter based on 10% validation set and calculate the F1 score on 10% testing set, then this is repeated for 5 times). We have included this details in the NeurIPS camera version. Feel free to let me know if you have any other question.
